src/components/swap/SwapLineItem.tsx

```diff
@@ -71,8 +71,8 @@ function ExchangeRateRow({ trade }: { trade: InterfaceTrade }) {
 }
 
 function ColoredPercentRow({ percent }: { percent: Percent }) {
-  const { formatPriceImpact } = useFormatter()
-  return <ColorWrapper textColor={getPriceImpactColor(percent)}>{formatPriceImpact(percent)}</ColorWrapper>
+  const { formatSlippage } = useFormatter()
+  return <ColorWrapper textColor={getPriceImpactColor(percent)}>{formatSlippage(percent)}</ColorWrapper>
```

Comment:
so there are other instances of price impact having the negative sign, including [here](https://github.com/Uniswap/interface/blob/main/src/components/swap/PriceImpactWarning.tsx#L44) 

i think we should just fix all price impact to be not negative sign since the negative sign never made sense to me lol

theres one place (the USD value change) where it would probably still make sense to keep, but we should special case that one

Reply:
Makes sense, I'll double check with product to make sure that they are aligned

Reply:
@tinaszheng Spoke to product, sounds like this is the only place we want to remove the minus sign